cloudhub/modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -249,6 +249,19 @@ image::cloudhub-deploy-app-config-tabs.png[Configuration tabs on the Deploy Appl
 
 The *Runtime* tab includes the following settings:
 
+Use Edge Release Channel::
+Starting with Mule 4.5, MuleSoft introduces two new release channels, xref:release-notes::mule-runtime/lts-edge-release-cadence.adoc[Edge and Long-term Support (LTS)].
++
+For new applications, the *Use Edge Release Channel* box is enabled by default. Uncheck this box if you want to use an LTS or a legacy runtime version.
```

Comment:
@luanamulesoft  lets remove all mentions of legacy. Please update the content to something similar to this https://docs.google.com/document/d/1P7Pw64oWo4dWwM6qvylR-50yKoW_HH2IARjvYje29mA/edit